src/classes/REL_Relationships_Con_TDTM.cls

```diff
@@ -85,6 +95,31 @@ public class REL_Relationships_Con_TDTM extends TDTM_Runnable {
             DmlWrapper w = createNewRelationships(ContactIds, ReciprocalIds, Types);
             dmlWrapper.objectsToInsert.addAll(w.objectsToInsert);
         }
+
+        if (!changedGender.isEmpty()) {
+            //query for a map of all relationships where where contact or related contact has changed gender
+            map<id,npe4__Relationship__c> mapRelationship = new map<Id,npe4__Relationship__c>(
+                [SELECT Id, npe4__Contact__c, npe4__RelatedContact__c, npe4__ReciprocalRelationship__c, npe4__Type__c 
+                FROM npe4__Relationship__c WHERE npe4__Contact__c IN :changedGender OR npe4__RelatedContact__c IN :changedGender]
+            );
+            //make a list of "Related Contact" lookup relationships in our set, these we want to update
+            list<npe4__Relationship__c> listRelsForUpdate = new list<npe4__Relationship__c>();
+            //make a list of mirror relationships for our first list, we need these for the addType method
+            list<npe4__Relationship__c> listMirrorRels = new list<npe4__Relationship__c>();
+
+            //get the relationships that the gender change affects
+            for (npe4__Relationship__c rel : mapRelationship.values()) {
+                if (changedGender.contains(rel.npe4__RelatedContact__c)) {
```

Comment:
if we are only looking at the RelatedContact, then why does the select on line 102 pick up rels for both npe4__Contact__c and npe4__RelatedContact__c?  don't you need to check both?